internal/sql/driver/sql_service.go

```diff
@@ -202,3 +204,14 @@ func ExtractTimeoutMillis(ctx context.Context) int64 {
 	}
 	return tomv.(int64)
 }
+
+func ExtractSchema(ctx context.Context) string {
+	if ctx == nil {
+		return DefaultSchema
+	}
+	v := ctx.Value(QuerySchemaKey{})
+	if v == nil {
+		return DefaultSchema
+	}
```

Comment:
If this is nil, we should send `null` schema to the cluster, should not we? I mean we should write a null frame to the client message. What will happen in current situation? 

I think empty string is not the same as null string regarding server behaviour.

Reply:
`string` in Go cannot be `nil`. `EncodeNullableForString` adds a null frame when a string is `""`, so this case is correctly handled.